apps/checkout-app/constants.ts

```diff
@@ -16,6 +21,8 @@ export const envVars: EnvVars = {
   appDomain: process.env.NEXT_PUBLIC_VERCEL_URL!,
   appUrl: `https://${process.env.NEXT_PUBLIC_VERCEL_URL!}`,
   apiUrl: process.env.NEXT_PUBLIC_SALEOR_API_URL!,
+  previewUrl: process.env.PREVIEW_URL!,
+  previewCheckoutToken: process.env.PREVIEW_CHECKOUT_TOKEN!,
```

Comment:
Same as preview url

Reply:
I'd wait for this functionality before we merge this in, the checkout will become obsolete after it's completed and it would require a change in the vars + redeployment to make it work again